.github/workflows/autobuild.yml

```diff
@@ -427,6 +430,25 @@ jobs:
           JAMULUS_BUILD_VERSION:    ${{ needs.create_release.outputs.build_version }}
           ARTIFACT_PATH:            deploy/${{ steps.get-artifacts.outputs.artifact_1 }}
 
+      ## RELEASE PROCEDURE FOR:
+      ## - Android Play Store - aab
+      ## Requirement: Service Account JSON setup:
+      ## - Google Play Console -> Setup -> API Access -> Create/Link Google Cloud Project 
+      ## - Google Cloud console -> IAM & Admin -> Service Accounts -> Create (Wizard). Then create JSON key and export/save
+      - name:                       Publish all Android ABI builds to Play Store
+        if:                         >-
+                                    needs.create_release.outputs.publish_to_release == 'true' &&
+                                    matrix.config.target_os == 'android'
+        id:                         publish_android
+        uses:                       r0adkll/upload-google-play@v1
```

Comment:
We need to review this action and confirm it's okay.

Reply:
I would rather check if it's possible like for iOS to use native cli tools or an official action by google.